modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -43,6 +44,34 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.13.2
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.2 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Improve Query to Clean up the Previews ===
+
+The database query used to identify the previews that need cleaning up has been enhanced for optimal performance. As a result, the background job responsible for this task will have significant performance improvements. https://github.com/owncloud/core/pull/40974[#40974]
+
+=== Fix View on Users Page for Subadmins ===
+
+Previously, subadmins couldn't read app config values. This caused problems in the users page because some of the functionalities were depending on the config values that subadmins couldn't read. These problems are now solved. https://github.com/owncloud/core/pull/40961[#40961]
+
+=== Delete All Files from Object Storage When User Is Deleted ===
+
+In previous versions when a user was deleted, files belonging to this user were not correctly removed from the object storage (s3) and were therefore left as remnants, unnecessarily using up space. This has been corrected and files are now properly removed. https://github.com/owncloud/core/pull/40959[#40959]
+
+=== Remove "Fill ETags" Repair Step ===
+
+The legacy "OC\Repair\FillETags" repair step, executed during upgrades, had prolonged execution times without delivering any further repairs. For this reason, the repair step has now been removed. https://github.com/owncloud/core/pull/40996[#40996]
+
+=== Allow Media Viewer App to Work with HEIC and HEIF Formats ===
+
+We removed the regular expression related to HEIC and HEIF files from the preview manager list as this was preventing the files_mediaviewer app from correctly interpreting these mime types. The app is now able to open such files. https://github.com/owncloud/core/pull/40990[#40990]
+
+=== Check if Account Creation Time Exists for Migrations ===
+
+In some rare scenarios it could have happened that the migration responsible for adding the creation time in the oc_accounts table was not correctly inserted into the oc_migrations one with the consequence that it was reattempted i.e. when upgrading apps, even if the column was already present. This has been now fixed. https://github.com/owncloud/core/pull/40991[#40991]
```

Comment:
```suggestion
In some rare scenarios it could have happened that the migration responsible for adding the creation time in the oc_accounts table was not correctly inserted into the oc_migrations one with the consequence that it was reattempted i.e. when upgrading apps, even if the column was already present. This has been fixed. https://github.com/owncloud/core/pull/40991[#40991]
```

"been now fixed" sounds strange, "now been fixed" is the usual word order. But I don't think that the word "now" adds any useful value, so may as well remove it.